src/output.js

```diff
@@ -0,0 +1,96 @@
+const { IncomingWebhook } = require('@slack/webhook')
+const { readFileSync } = require('fs')
+
+/**
+ * @param {{type: string, status: number, slackWebhookUrl: string}} params
+ */
+async function run({ type, ...args }) {
+  try {
+    if (type === 'github') {
+      // tbd
+      // await githubCheck(args)
```

Comment:
Github Check should be set on every run (also, option to disable the behavior). Still, Github Message should be the same as Slack message (only on error when a developer's attention needed.

Reply:
Github check as in an additional check to send, i.e. what Paul was working on in #2 ? Should that be in additional PR, and leave this as a placeholder?

Reply:
Yup, it's kinda dogfooding that it will be here as well :)

Reply:
We can proceed in additional PR :)

Reply:
I think if that's the case, then remove L#10 & 11, and make it:
```javascript
    if (type === 'slack') {
      await slackNotification(args)
    } else {
      console.log('Unknown output type: ', type)
    }
```
Then add more types as they come up 🤷‍♂